Mathlib/Tactic/MPTests.lean

```diff
@@ -0,0 +1,297 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+
+import Std.Data.String.Basic
+import Std.Lean.Expr
+import Std.Lean.PersistentHashMap
+import Std.Lean.Syntax
+import Mathlib.Tactic.DefEqTransformations
+import Mathlib.Tactic.Lemma
+import Mathlib.Tactic.Set
+
+/-!
+#  A testing linter
+
+In a file with `import Mathlib.Tactic.MPTests`, you can run the tests in a single command
+writing `#test cmd`.
```

Comment:
Needs a more specific name.

Reply:
`#test_for_meta_bugs`?

Reply:
I renamed the linter `metaTest` and the `#`-commands now involve the word `meta`.